backports/ignition-cmake2/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: Kazuki Takahashi <ktakahashi@seqsense.com>
+# Maintainer:
+pkgname=ignition-cmake2
+pkgver=2.17.1
+pkgrel=1
+pkgdesc="A set of CMake modules that are used by the C++-based Gazebo projects."
+url="https://github.com/gazebosim/gz-cmake"
+arch="all"
+license="Apache-2.0"
+depends="cmake"
+makedepends="cmake"
+checkdepends=""
+install=""
+subpackages="$pkgname-dev $pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/gazebosim/gz-cmake/archive/refs/tags/ignition-cmake2_$pkgver.tar.gz"
```

Comment:
These archives seem containing full source of https://github.com/gazebosim/gz-math, so I think both packages contain full `gz-math` now

Reply:
Nevermind, these archives seem specialized for each subpackage (directories like examples and tutorials are in both of them)

Reply:
```
$ tar tzf ignition-cmake2-2.17.1-r1.apk 2> /dev/null 
.SIGN.RSA.-65af7309.rsa.pub
.PKGINFO
usr/
usr/share/
usr/share/cmake/
usr/share/cmake/ignition-cmake2/
usr/share/cmake/ignition-cmake2/cmake2/
usr/share/cmake/ignition-cmake2/cmake2/Export.hh.in
usr/share/cmake/ignition-cmake2/cmake2/FindAVCODEC.cmake
usr/share/cmake/ignition-cmake2/cmake2/FindAVDEVICE.cmake
usr/share/cmake/ignition-cmake2/cmake2/FindAVFORMAT.cmake
usr/share/cmake/ignition-cmake2/cmake2/FindAVUTIL.cmake
usr/share/cmake/ignition-cmake2/cmake2/FindCPPZMQ.cmake
usr/share/cmake/ignition-cmake2/cmake2/FindDL.cmake
usr/share/cmake/ignition-cmake2/cmake2/FindEIGEN3.cmake
usr/share/cmake/ignition-cmake2/cmake2/FindFreeImage.cmake
usr/share/cmake/ignition-cmake2/cmake2/FindGTS.cmake
usr/share/cmake/ignition-cmake2/cmake2/FindIFADDRS.cmake
usr/share/cmake/ignition-cmake2/cmake2/FindIgnBullet.cmake
usr/share/cmake/ignition-cmake2/cmake2/FindIgnCURL.cmake
usr/share/cmake/ignition-cmake2/cmake2/FindIgnOGRE.cmake
usr/share/cmake/ignition-cmake2/cmake2/FindIgnOGRE2.cmake
usr/share/cmake/ignition-cmake2/cmake2/FindIgnProtobuf.cmake
usr/share/cmake/ignition-cmake2/cmake2/FindIgnURDFDOM.cmake
usr/share/cmake/ignition-cmake2/cmake2/FindJSONCPP.cmake
usr/share/cmake/ignition-cmake2/cmake2/FindODE.cmake
usr/share/cmake/ignition-cmake2/cmake2/FindOptiX.cmake
usr/share/cmake/ignition-cmake2/cmake2/FindSQLite3.cmake
usr/share/cmake/ignition-cmake2/cmake2/FindSWSCALE.cmake
usr/share/cmake/ignition-cmake2/cmake2/FindTINYXML2.cmake
usr/share/cmake/ignition-cmake2/cmake2/FindUUID.cmake
usr/share/cmake/ignition-cmake2/cmake2/FindYAML.cmake
usr/share/cmake/ignition-cmake2/cmake2/FindZIP.cmake
usr/share/cmake/ignition-cmake2/cmake2/FindZeroMQ.cmake
usr/share/cmake/ignition-cmake2/cmake2/IgnBenchmark.cmake
usr/share/cmake/ignition-cmake2/cmake2/IgnCMake.cmake
usr/share/cmake/ignition-cmake2/cmake2/IgnCheckSSE.cmake
usr/share/cmake/ignition-cmake2/cmake2/IgnCodeCheck.cmake
usr/share/cmake/ignition-cmake2/cmake2/IgnCodeCoverage.cmake
usr/share/cmake/ignition-cmake2/cmake2/IgnConfigureBuild.cmake
usr/share/cmake/ignition-cmake2/cmake2/IgnConfigureProject.cmake
usr/share/cmake/ignition-cmake2/cmake2/IgnCreateDocs.cmake
usr/share/cmake/ignition-cmake2/cmake2/IgnGenerateVersionInfo.cmake
usr/share/cmake/ignition-cmake2/cmake2/IgnImportTarget.cmake
usr/share/cmake/ignition-cmake2/cmake2/IgnManualSearch.cmake
usr/share/cmake/ignition-cmake2/cmake2/IgnPackaging.cmake
usr/share/cmake/ignition-cmake2/cmake2/IgnPkgConfig.cmake
usr/share/cmake/ignition-cmake2/cmake2/IgnPython.cmake
usr/share/cmake/ignition-cmake2/cmake2/IgnRonn2Man.cmake
usr/share/cmake/ignition-cmake2/cmake2/IgnSanitizers.cmake
usr/share/cmake/ignition-cmake2/cmake2/IgnSetCompilerFlags.cmake
usr/share/cmake/ignition-cmake2/cmake2/IgnUtils.cmake
usr/share/cmake/ignition-cmake2/cmake2/JoinPaths.cmake
usr/share/cmake/ignition-cmake2/cmake2/cmake_uninstall.cmake.in
usr/share/cmake/ignition-cmake2/cmake2/cpack_options.cmake.in
usr/share/cmake/ignition-cmake2/cmake2/ign_auto_headers.hh.in
usr/share/cmake/ignition-cmake2/cmake2/ignition-all-config.cmake.in
usr/share/cmake/ignition-cmake2/cmake2/ignition-component-config.cmake.in
usr/share/cmake/ignition-cmake2/cmake2/ignition-config.cmake.in
usr/share/cmake/ignition-cmake2/cmake2/pkgconfig/
usr/share/cmake/ignition-cmake2/cmake2/pkgconfig/ignition-component.pc.in
usr/share/cmake/ignition-cmake2/cmake2/pkgconfig/ignition.pc.in
usr/share/cmake/ignition-cmake2/cmake2/upload_doc.sh.in
usr/share/cmake/ignition-cmake2/cmake2/version_info.json.in
usr/share/cmake/ignition-cmake2/ignition-cmake2-config-version.cmake
usr/share/cmake/ignition-cmake2/ignition-cmake2-config.cmake
usr/share/cmake/ignition-cmake2/ignition-cmake2-utilities-targets.cmake
usr/share/ignition/
usr/share/ignition/ignition-cmake2/
usr/share/ignition/ignition-cmake2/benchmark/
usr/share/ignition/ignition-cmake2/benchmark/run_benchmarks.py
usr/share/ignition/ignition-cmake2/codecheck/
usr/share/ignition/ignition-cmake2/codecheck/cpplint.py
usr/share/ignition/ignition-cmake2/codecheck/header_guard.rule
usr/share/ignition/ignition-cmake2/codecheck/namespace_AZ.rule
usr/share/ignition/ignition-cmake2/doxygen/
usr/share/ignition/ignition-cmake2/doxygen/api.in
usr/share/ignition/ignition-cmake2/doxygen/cppreference.tag.xml
usr/share/ignition/ignition-cmake2/doxygen/footer.html
usr/share/ignition/ignition-cmake2/doxygen/header.html
usr/share/ignition/ignition-cmake2/doxygen/layout.xml
usr/share/ignition/ignition-cmake2/tools/
usr/share/ignition/ignition-cmake2/tools/check_test_ran.py
usr/share/ignition/ignition-cmake2/tools/doc_check.sh

$ tar tzf ignition-math6-6.15.0-r1.apk 2> /dev/null 
.SIGN.RSA.-65af7309.rsa.pub
.PKGINFO
usr/
usr/lib/
usr/lib/libignition-math6.so.6
usr/lib/libignition-math6.so.6.15.0
```
Contents looks fine